SetupDataPkg/Tools/VariableList.py

```diff
@@ -3,7 +3,6 @@
 # Copyright (c) 2022, Microsoft Corporation. All rights reserved.<BR>
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 #
-

```

Comment:
It looks like you removed a lot of white space when you were attempting to resolve flake8 errors. Can you add back the white space that didn't need to be removed?